### DISCUSSION TITLE
thumbnail on desktop screen in windows 11

### DISCUSSION THREAD

**Discussion by avijtt** [sumatrapdfreader/sumatrapdf#3296] [Q&A]
I recently encountered when a pdf file on desktop screen shows only plane icon/thumbnail instead of pdf thumbnail of sumatra pdf . Is there any way to change the thumbnail?
![Screenshot 2023-02-22 215218](https://user-images.githubusercontent.com/100214776/220684888-b917bdfa-be65-474e-8747-564cf82c01ba.png)

**Comment by GitHubRulesOK** [comment]
should be a windows setting
![image](https://user-images.githubusercontent.com/12754913/220711515-33a3c0df-2f98-40ed-a269-7201d7de326c.png)

But the effect is variable

![image](https://user-images.githubusercontent.com/12754913/220711730-71a6b0fc-5fbc-4175-91a6-d0129b4a9a92.png)

![image](https://user-images.githubusercontent.com/12754913/220712317-ac4eadf9-cc30-440e-a657-5e18ec0e48a1.png)

**Comment by DanteCampos** [comment]
Same problem here, setting folder options didn't work

A screenshot from my desktop:
![image](https://user-images.githubusercontent.com/41027230/221487129-0787524f-1ed6-40bd-adb1-84b64a1a79bb.png)

**Comment by GitHubRulesOK** [comment]
@DanteCampos  your showing a different setting

Depending on OS and SumatraPDF version the default style of icons may differ. To be clearer, 

When Previewer is installed
---
Thumbnails OFF Icons OFF
whatever the system decides

Thumbnails OFF Icons ON (default icon in use may vary)
![image](https://user-images.githubusercontent.com/12754913/221573088-da4e2e87-b338-4e05-84c4-f772140e823d.png)

Thumbnails ON Icons OFF (the files that cannot be previewed locked or broken keep their default Icons)
![image](https://user-images.githubusercontent.com/12754913/221573333-d781defd-2910-41e1-9b20-a2499df1e815.png)

Thumbnails ON with icons ON the question as raised at the start
![image](https://user-images.githubusercontent.com/12754913/221573715-a263c578-b868-4761-b11b-b956b933e53c.png)

**Reply by DanteCampos** [reply]
Thanks, I reinstalled Sumatra and now the OS settings are working
![image](https://user-images.githubusercontent.com/41027230/221814012-e0d11b04-f0ca-42ee-a835-2ed6cb89739c.png)

**Comment by rudolphos** [comment]
Doesn't work for me. SumatraPDF doesn't generate thumbnails. I've reinstalled it several times with _thumbnails_ generation box checked.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/4366810/d53d1d6d-969a-43f1-afd7-37c8d113a08b)

**Reply by GitHubRulesOK** [reply]
hmm the setting is Not *"thumbnails"* it is ensure preview is installed
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2128d657-2131-4a10-b13c-8b5c0b8009ee)


one factor that may make a difference is 64 system needs 64 bit install not 32 bit install, another possibility is some security setting may interfere, but personally never had problems. The file content must be valid and not be locked with a password to open, and there should not be any other previewer previously installed such as Acrobat, that may block previewer activity.

**Reply by rudolphos** [reply]
I usually install latest pre-release that is 64bit. And I have a folder with 40 pdf files, none of them are locked. I don't have acrobat but I do have Foxit. After your suggestion, I reinstalled Sumatra once more time. With checkbox checked: 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/4366810/a35d0490-da16-46e4-a10a-85e6a44ce4cf)
And it doesn't work:
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/4366810/423a0393-f1e4-4774-817f-fa963fd9e4db)

**Reply by GitHubRulesOK** [reply]
hmm lets see
@rudolphos do verify your system if you can run 64bit installer then use that one it looks like your using a 32bit variant ??
THIS IS a 32 bit system
Acrobat active
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2852168f-daa2-46c7-b685-2d3c2f0771b8)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/a369bc17-48f7-4d48-92a4-670c58eede82)

change status
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4b3a21d0-153d-4ab0-92b8-31a6f9c409ff)
Acrobat is previewing on right but not switching to thumbnails
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/b1289264-f6b8-4a0a-9134-43ada7b6f5df)

32bit install BECAUSE THIS IS 32bit SYSTEM
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5363c4b0-c8e4-4c43-84ce-274c3e1d0574)

32bit previews on 32bit system
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c775d18d-0cbf-4cd9-950f-0690835d9c1b)
